### PR TITLE
feat(issue-1): finish show prefix highlight

### LINKED ISSUE #1: no prefix highlight in tmux
show prefix highlight in tmux

### PR DESCRIPTION
**Pull request by jisung-lee7**
maybe you forgot

### CODE REVIEW

File: .tmux.conf [hunk 1]
```diff
@@ -90,7 +90,7 @@ wg_activated_gh_account="#(~/.config/tmux/.__activated_gh_account.sh)"
 wg_working_dir="#(~/.config/tmux/.__shorten_current_path.sh #{pane_current_path})"
 
 set -g status-left "$wg_session"
-set -g status-right "#[fg=$color_secondary]$wg_activated_gh_account #[default]| #[fg=$color_working_dir]$wg_working_dir #[default]| CPU: #{cpu_fg_color}#{cpu_percentage} #[default]| RAM: #{ram_fg_color}#{ram_percentage}  "
+set -g status-right "#{prefix_highlight} #[fg=$color_secondary]$wg_activated_gh_account #[default]| #[fg=$color_working_dir]$wg_working_dir #[default]| CPU: #{cpu_fg_color}#{cpu_percentage} #[default]| RAM: #{ram_fg_color}#{ram_percentage}  "
```
Comment (jiyeol-lee) [inline .tmux.conf:93]: what is this purpose?

Comment (jisung-lee7) [reply .tmux.conf:93]: image
I thought you accidentally deleted that along with the battery settings.
If that is not the case, you don't need to add it.😂

Comment (jiyeol-lee) [reply .tmux.conf:93]: oh, you are right. it's a mistake. thank you for the update. feel free to merge

**Review by jiyeol-lee — APPROVED** [review]

File: .tmux.conf (hunk 1 above)
Comment (jisung-lee7) [reply .tmux.conf:93]: 😀 Good. Then, can I get write access? I think I'm blocked from merge.....
image

**Pull request merged**